runner_facebook.sh

```diff
@@ -1,23 +1,25 @@
 #!/usr/bin/env bash
```

Comment:
This needs to be changed, since it will not work in all operating system environments

Reply:
This was modified locally to /usr/local/bin.

A means to determine which $PATH is appropriate for OS is needed.